### DISCUSSION TITLE
Support hashids/hashids 5.0

### DISCUSSION THREAD

**Discussion by terranc** [deligoez/laravel-model-hashid#22] [Ideas]
Because laravel-hashids is used in laravel, and only laravel-hashids v11 supports laravel 10, it uses hashids v5.0 internally, which leads to conflicts. I hope to update composer.json

**Comment by deligoez** [comment]
This is resolved in **v4.0.0** — our `composer.json` now requires `"hashids/hashids": "^4.1|^5.0"`, so both versions are supported.

Please upgrade to v4 to resolve the dependency conflict.
